admin/app.js

```diff
@@ -273,6 +274,7 @@ if (WEBSITE_OFFLINE) {
   app.use('/', index)
   app.use('/test-developer', testDeveloper)
   app.use('/service-manager', serviceManager)
+  app.use('/service-manager/service-message', serviceMessage)
```

Comment:
wondering if instead I should use the service manager route and include the service message controller within.

Reply:
Wondering why it needs to be under `/service-manager/`?

Reply:
it's a service manager related feature

Reply:
There is no technical requirement to organise routes by role.  Furthermore, the user is unlikely to require it on a system with RBAC.  It leads to large files and a maintenance headache.

So why follow a pattern that has so little value?

Reply:
@jon-shipley done.